wiki/Tournaments/MXB/3/en.md

```diff
@@ -0,0 +1,159 @@
+---
+tags:
+- MXB
+---
+# The 3rd MXB osu!mania 4K Tournament
```

Comment:
Use consistent tournament names. Assuming this is the right one from looking at the logo.
```suggestion
# Novice Mania Cup 3rd
```